src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -162,6 +153,10 @@ BinaryCompactObject::BinaryCompactObject(
                 "Must specify either both inner and outer boundary conditions "
                 "or neither.");
   }
+  if (envelope_radius_ >= outer_radius_) {
+    PARSE_ERROR(context,
+                "The outer radius must be larger than the envelope radius.");
```

Comment:
Do we also have a check that the `envelope_radius` must be larger than sqrt(3) * L? (L being the length of the object cube `length_inner_cube_`)

Reply:
We have `length_outer_cube_ > 2.0 * length_inner_cube_` (line 112). Is that correct?

Reply:
that works!